### PR TITLE
Fix pickup.list being empty

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.API/Features/Pickups/Pickup.cs [hunk 1]
```diff
@@ -56,8 +56,8 @@ internal Pickup(ItemPickupBase pickupBase)
             // prevent prefabs like `InventoryItemLoader.AvailableItems[ItemType.GrenadeHE].PickupDropModel` from adding to pickup list
             if (pickupBase.Info.ItemId is ItemType.None)
                 return;
-
-            BaseToPickup.Add(pickupBase, this);
+            if (!BaseToPickup.ContainsKey(pickupBase))
+                BaseToPickup.Add(pickupBase, this);
```
Comment (IRacle1) [inline Exiled.API/Features/Pickups/Pickup.cs:60]: Im pretty sure it is useless, for getting/creating pickups we use `Pickup::Get`, who has that check

File: Exiled.Events/Events.cs [hunk 2]
```diff
@@ -101,7 +101,8 @@ public override void OnEnabled()
 
             RagdollManager.OnRagdollSpawned += Handlers.Internal.RagdollList.OnSpawnedRagdoll;
             RagdollManager.OnRagdollRemoved += Handlers.Internal.RagdollList.OnRemovedRagdoll;
-
+            ItemPickupBase.OnPickupAdded += x => Pickup.BaseToPickup.Add(x, new Pickup(x));
```
Comment (IRacle1) [inline Exiled.Events/Events.cs:104]: dont use `new Pickup(x)` use `Pickup::Get`

**Review by IRacle1 — CHANGES_REQUESTED** [review]
Idk if that change can break anything or not, our PickupAPI is very complex

File: Exiled.Events/Events.cs [hunk 3]
```diff
@@ -101,7 +101,8 @@ public override void OnEnabled()
 
             RagdollManager.OnRagdollSpawned += Handlers.Internal.RagdollList.OnSpawnedRagdoll;
             RagdollManager.OnRagdollRemoved += Handlers.Internal.RagdollList.OnRemovedRagdoll;
-
+            ItemPickupBase.OnPickupAdded += x => Pickup.BaseToPickup.Add(x, Pickup.Get(x));
```
Comment (IRacle1) [inline Exiled.Events/Events.cs:104]: oh, bruh, i forgot that `Pickup.Get(x)` includes adding to list, just keep `Pickup.Get` and remove `Pickup.BaseToPickup.Add`

**Review by IRacle1 — CHANGES_REQUESTED** [review]

File: Exiled.Events/Events.cs (hunk 3 above)
Comment (IRacle1) [reply Exiled.Events/Events.cs:104]:
```suggestion
            ItemPickupBase.OnPickupAdded += x => Pickup.Get(x);
```

**Review by IRacle1 — APPROVED** [review]

**Comment by louis1706** [comment]
@joker-119

**Comment by joker-119** [comment]
I am failing to see how this would possibly fix the Pickup.List being empty, when you are removing the only line of code that adds to the dictionary for pickups.

**Pull request merged**